src/MiniCover/Commands/Options/IMiniCoverOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+
+namespace MiniCover.Commands.Options
+{
+    internal interface IMiniCoverOption
+    {
+        string OptionTemplate { get; }
```

Comment:
Why are those properties in this interface?

Reply:
To enforce template on developer.